backend/Watchdog.Core/Watchdog.Core.BLL/Services/MemberService.cs

```diff
@@ -1,30 +1,94 @@
 ﻿using AutoMapper;
 using Microsoft.EntityFrameworkCore;
+using SendGrid;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Watchdog.Core.BLL.Models;
 using Watchdog.Core.BLL.Services.Abstract;
-using Watchdog.Core.Common.DTO.Member;
+using Watchdog.Core.Common.DTO.Members;
 using Watchdog.Core.DAL.Context;
 using Watchdog.Core.DAL.Entities;
 
 namespace Watchdog.Core.BLL.Services
 {
+
     public class MemberService : BaseService, IMemberService
     {
-        public MemberService(WatchdogCoreContext context, IMapper mapper) : base(context, mapper)
+        private readonly IEmailSendService _emailSendService;
+
+        public MemberService(WatchdogCoreContext context, IMapper mapper, IEmailSendService emailSendService) : base(context, mapper)
         {
+            _emailSendService = emailSendService;
         }
 
-        public async Task<ICollection<MemberDto>> GetMembersByOrganizationIdAsync(int id)
+        public async Task<Response> InviteMemberAsync(int id)
         {
-            return _mapper.Map<ICollection<MemberDto>>(await _context.Members.Where(m => m.OrganizationId == id).Include(m => m.User).ToListAsync());
+            var member = await _context.Members
+                .Include(m => m.User)
+                .FirstOrDefaultAsync(m => m.Id == id) ?? throw new KeyNotFoundException("Member doesn't exist");
+
+            return await InviteMemberAsync(_mapper.Map<MemberDto>(member));
         }
 
-        public async Task<ICollection<MemberDto>> GetAllMembersAsync()
+        public Task<Response> InviteMemberAsync(MemberDto memberDto)
+        {
+            ExampleTemplateData data = new()
```

Comment:
see comment above
The same for  ExampleTemplateData

Reply:
It is not my class too. It is example class for SendEmailService. We don't have template for send invitations letters yet, so I have used ExapleTemplate and it will be changed in future on real model.